src/App.tsx

```diff
@@ -60,6 +61,28 @@ const App = ({ skipLoadingScreen }: AppProps) => {
     ]);
   };
 
+  const theme = extendTheme({
+    // This is for the Dark theme that the user chooses
+    components: {
+      Text: {
+        // For all the Text components
+        baseStyle: ({ colorMode }) => {
+          return {
+            color: colorMode === 'light' ? '#333' : '#fff',
+          };
+        },
+      },
+      View: {
+        baseStyle: ({ colorMode }: any) => {
+          return {
+            flex: 1,
+            bg: colorMode === 'light' ? '#fff' : '#333',
+          };
+        },
+      },
+    },
+  });
```

Comment:
We should move this outside `src/App.tsx` to some more appropriate place like `src/themes/index.ts` where you can set up everything themes related and here you can just import the theme.

Reply:
Also, make sure we are referencing native-base color constants instead of hardcoded color hex values.